### DISCUSSION TITLE
#3511 讨论

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#3650] [讨论]
#3511

**Comment by ailelix** [comment]
很抱歉下午忘记看邮箱和Github了
这边补一个现录的Log
可以确认应该是`MaxDirectMemory`参数过小导致的内存耗尽
这边感谢龙猫和各位如此迅速的回复和处理
[crash-2024-04-09_00.39.03-client.txt](https://github.com/Hex-Dragon/PCL2/files/14908541/crash-2024-04-09_00.39.03-client.txt)

**Reply by ailelix** [reply]
或许可以考虑根据JDK与JVM版本配置不同的默认参数？（仅猜想）

**Reply by wuliaodexiaoluo** [reply]
被龙猫标记为破坏性改动的参数估计是不太可能再加进去了

**Reply by wuliaodexiaoluo** [reply]
这个看起来还是给太小了
要求268M以上。。。
```
java.lang.OutOfMemoryError: Cannot reserve 16777216 bytes of direct buffer memory (allocated: 265556319, limit: 268435456)
```
